src/player.py

```diff
@@ -16,4 +16,11 @@ def __init__(self):
     self.ffa_stats = Stats()
 
   def print(self):
+    if self.solo_stats.wins == -1 or self.team_stats.wins == -1 or self.ffa_stats.wins == -1:
```

Comment:
If you take my suggestion above of setting team / ffa stats to -1 when scraping from warcraft3.info, change this from `or` to `and`